contracts/root/predicates/ERC20Predicate.sol

```diff
@@ -433,8 +441,8 @@ contract ERC20Predicate is IErcPredicate {
         "Withdrawer and referenced tx do not match"
       );
       closingBalance = BytesLib.toUint(logData, 64); // output1
-    } else if (eventSignature == LOG_TRANSFER_EVENT_SIG) {
-      // event LogTransfer(
+    } else if (eventSignature == LOG_TRANSFER_EVENT_SIG || eventSignature == LOG_FEE_TRANSFER_EVENT_SIG) {
```

Comment:
`LOG_FEE_TRANSFER_EVENT_SIG` sig is present in each and every transaction !?

Reply:
After the Bor changes, yes.